coalib/bearlib/languages/definitions/Ruby.py

```diff
@@ -3,4 +3,9 @@
 
 @Language
 class Ruby:
-    pass
+    aliases = 'rb',
+    extensions = '.rb',
+    comment_delimiter = '#'
+    multiline_comment_delimiters = {'=begin': '=end'}
+    string_delimiters = {'"': '"', "'": "'", '%q(': ')', '%Q(': ')', '%(': ')'}
```

Comment:
`%( )` is not correct here.

As mentioned on https://github.com/coala/coala/issues/5572 , Ruby was selected because it uses balanced string literals.

i.e. in `%( ... ( .... ) ...)` , the first `)` is not the end of the string.

We dont have a property for balanced delimiters yet, and we need the Ruby language file to exist in order to add that feature to our Ruby lang file, along with others.

Reply:
Still not understand it. Is removin the `%()` string delimiters from it, will solve?